### PR TITLE
Fix export of EMF models using single containments

### LINKED ISSUE #111: Single Containment reference is not supported
Trying to export a model that uses a meta model that has a reference set as a single containment(Lower & Upper bound=1) throws and unsupported exception. [Example of code with this problem.](https://github.com/LionWeb-io/property-demo/tree/atanas/single-containment-bug)
[Line in EMFModelExporter where problem occurs.](https://github.com/LionWeb-io/lionweb-java/blob/f6a67bdd1a7dd54c032dedb9a95d0458cdc4ad25/emf/src/main/java/io/lionweb/lioncore/java/emf/EMFModelExporter.java#L67)
## How to reproduce bug:
In property-demo repository execute
`./gradlew xtend-generators:emf-generator:run`

### PR DESCRIPTION
**Pull request by ftomassetti**
Fix #111

### CODE REVIEW

File: emf/src/main/java/io/lionweb/lioncore/java/emf/EMFModelExporter.java [hunk 1]
```diff
@@ -64,7 +64,17 @@ public EObject exportTree(Node root, ReferencesPostponer referencesPostponer) {
                             .collect(Collectors.toList());
                     eObject.eSet(eReference, childrenInEmf);
                   } else {
-                    throw new UnsupportedOperationException();
+                    List<? extends Node> childrenInLW =
+                        root.getChildrenByContainmentName(eReference.getName());
+                    if (childrenInLW.size() > 1) {
+                      throw new IllegalStateException(
+                          "More than one child found, where up to one ");
```
Comment (enikao) [inline emf/src/main/java/io/lionweb/lioncore/java/emf/EMFModelExporter.java:71]: I think this error message is hard to understand without the context.

File: emf/src/main/java/io/lionweb/lioncore/java/emf/mapping/ConceptsToEClassesMapping.java [hunk 2]
```diff
@@ -136,7 +136,11 @@ public boolean knows(EClassifier eClassifier) {
     return null;
   }
 
-  public void populateInternal() {
+  public void prePopulateBuiltins() {
+    ePackagesToLanguages.put(
+        BuiltinsEMFPackageProvider.getEPackage(), LionCoreBuiltins.getInstance());
```
Comment (enikao) [inline emf/src/main/java/io/lionweb/lioncore/java/emf/mapping/ConceptsToEClassesMapping.java:141]: What does `BuiltinsEMFPackageProvider` do?

Comment (ftomassetti) [reply emf/src/main/java/io/lionweb/lioncore/java/emf/mapping/ConceptsToEClassesMapping.java]: I should have removed that. This was the thing I was using before getting the Builtins EMF Package from your PR. Removed

File: emf/src/main/java/io/lionweb/lioncore/java/emf/EMFModelExporter.java (hunk 1 above)
Comment (ftomassetti) [reply emf/src/main/java/io/lionweb/lioncore/java/emf/EMFModelExporter.java]: Right, I tried updating it

**Review by enikao — APPROVED** [review]

**Comment by ftomassetti** [comment]
Thank you @enikao , I will do a new release right now (0.1.4) so you can use these fixes more easily

**Pull request merged**